joplin/css/editor.scss

```diff
@@ -65,3 +68,7 @@ label {
 .sequence-container.sequence-type-list {
     width: 100% !important;
 }
+
+.odd-placeolder-elm {
+  color: $coa-color-dark-gray2 !important;
```

Comment:
Is `!important` required?

Reply:
I need to watch that! 🤦‍♂️Past experience created a bad habit there! AND, to mention/note-out why it's needed if so. 
- In this case it turns out it is required. Looks like Drafttail's editor classes render before our `editor.scss` style. At least that's my understanding.

Reply:
All good then!